packages/charts-bar/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@ibm-design/charts-bar",
+  "version": "0.1.0",
+  "description": "Sample implementation of the charts bar from the IBM Design Language",
+  "author": "Abby Starnes & Stephane Rodet",
```

Comment:
Indeed, only contributors can be an array though. I'll think about that and what makes most sense, and also look on the other projects.